src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -39,6 +39,7 @@ protected function configure(): void
             ->setDefinition([
                 new InputArgument('id', InputArgument::OPTIONAL, 'Specific message id to show'),
                 new InputOption('max', null, InputOption::VALUE_REQUIRED, 'Maximum number of messages to list', 50),
+                new InputOption('failure-transport', null, InputOption::VALUE_OPTIONAL, 'Use a specific failure transport'),
```

Comment:
You have`VALUE_OPTIONAL`. That means I can pass simply `--failure-transport=` with no value. We should make this `VALUE_REQUIRED`. Another option would be to interactively ask which transport they want if they pass no value.

Reply:
is it ok for BC to make it REQUIRED?